index.js

```diff
@@ -40,6 +41,7 @@ module.exports = function createStore (reducer, initialState) {
     dispatch: dispatch,
     subscribe: subscribe,
     getState: getState,
-    replaceReducer: replaceReducer
+    replaceReducer: replaceReducer,
+    combineReducers: combineReducers
```

Comment:
@sethvincent is `combineReducers` meant to be returned by `createStore`? it's [not done in redux](https://github.com/reactjs/redux/blob/master/src/createStore.js).